config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<!-- This is a checkstyle configuration file. For descriptions of
```

Comment:
added in a few additional checks.  Import order fails... it would be easy to fix however we should have a conversation around it.

added issue: https://github.com/mesosphere/hdfs/issues/97

all other checkstyles added